jekyll/_docs/fork-pr-builds.md

```diff
@@ -1,105 +1,43 @@
-   PRs.
+   These are passphraseless private keys which you can add to CircleCI to access arbitrary hosts during a build. They are not made available to fork PR builds.
 
-   <span class='label label-info'>Note:</span> If the same key is set in both, the web UI value overrides the circle.yml one.
+1. **AWS permissions (Settings > Permissions > AWS Permissions)**
 
-2. **Checkout keys**
-   (configured in Project settings > Permissions > Checkout SSH keys)
+   This configuration gets stored in files in `~/.aws/` and would allow hostile code to perform actions through AWS API calls. This file is not made available to the fork PR builds.
 
-   In normal circumstances, we use either a per project deploy key or a GitHub user key to check
-   out the code during a build. Deploy keys are keys valid for a specific project only, while a user
-   key can be used to act as that user on GitHub.
+1. **Deployment Credentials**
 
-   For safe fork PR builds, we use a user key associated with a machine GitHub user that
-   CircleCI controls, so if they key is leaked due to a malicious PR it has no impact
-   on your project.
+   Currently we offer streamlined configuration to facilitate deployment to Heroku and CodeDeploy. Any credentials you configure here will not be made available to fork PR builds.
 
-3. **SSH keys**
-   (configured in Project settings > Permissions > SSH Permissions)
-
-   These are passphraseless private keys which you can add to CircleCI to access arbitrary
-   hosts during a build. They are not made available to fork PR builds.
-
-4. **AWS permissions**
-   (configured in Project settings > Permissions > AWS Permissions)
-
-   This configuration gets stored in files in ~/.aws/ and would allow hostile code to perform actions through AWS API calls. This file is not made available to the fork PR builds.
-
-5. **Deployment credentials**
-
-   Currently we offer streamlined configuration to facilitate deployment to Heroku and
-   CodeDeploy. Any credentials you configure here will not be made available to fork PR builds.
-
-#### CircleCI will only run builds of pull requests from forks for public GitHub repositories by default.
-
-We do this because of the **Checkout keys** issue above; we can't use a generic user key to
-check out your code for a private repository.
-
-
-### Unsafe fork PR builds
-
-In the event that you *want* these five categories of configuration to be made available, or you need
-to run fork PR builds for private repositories, you need to affirmatively enable it because of these
-security issues.
-
-There is a per-project flag (in Project settings > Build Settings > Advanced Settings > Project fork pull requests)
-which will cause us to run builds of all fork pull requests without suppressing any of the sensitive
-information listed above.
-
-### Note
-
-CircleCI currently does not support this feature for Bitbucket repositories. 
+*Note: CircleCI currently does not support this feature for Bitbucket repositories.*
```

Comment:
This is correct, we don’t yet support building fork pull requests for Bitbucket.